packages/odyssey-storybook/package.json

```diff
@@ -13,14 +13,17 @@
     "url": "https://github.com/okta/odyssey",
     "directory": "packages/odyssey-storybook"
   },
+  "dependencies": {
+    "@faker-js/faker": "^8.2.0",
+    "@okta/odyssey-react-mui": "workspace:*"
```

Comment:
Why is this moved up to `dependencies`?
Should `faker` also be a devDependency?